libnd4j/UnderstandingGraph.md

```diff
@@ -37,37 +40,6 @@ There are some limitations. Some of them will be lifted eventually, others won't
 - CUDA isn't supported at this moment. _This limitation will be lifted soon._
 - When used from C++, Graph only supports FeedForward mode. _This limitation will be lifted soon._
 
-### Minified Graph binaries
-There's an option to build minified binaries suited for execution of ***specific graphs***. Idea is quite simple: you feed your existing Graph(s) in FlatBuffers format into special app, which extracts operations used in your Graph(s) and excludes all other operations from target binary.
```

Comment:
@treo removed this since it was out of date and the functionality was moved to java/bash level.